sympy/solvers/bivariate.py

```diff
@@ -113,6 +114,28 @@ def _linab(arg, symbol):
         x = -x
     return a, b, x
 
+def power_list(f, *gens, **args):
```

Comment:
Document `f`, `gens` and `args`

Reply:
Okay will do in next commits after changing powerlist